client/ionic/public/index.html

```diff
@@ -21,6 +21,10 @@
     <meta name="apple-mobile-web-app-capable" content="yes" />
     <meta name="apple-mobile-web-app-title" content="Ionic App" />
     <meta name="apple-mobile-web-app-status-bar-style" content="black" />
+
+    <!-- css stylesheet -->
+    <link rel="stylesheet" href="https://unpkg.com/tachyons@4.10.0/css/tachyons.min.css"/>
```

Comment:
all modules should be sourced solely locally

Reply:
done, brought it in via npm